errors/version-staleness.md

```diff
@@ -0,0 +1,60 @@
+# Version Staleness
+
+#### Why This Error Occurred
+
+In the error overlay, a message was shown that the detected Next.js version was out-of-date.
+
+To get the newest features and bug fixes, it is recommended to stay up to date.
+
+#### Possible Ways to Fix It
+
+If you are testing out a canary release, upgrade Next.js with one of the following:
+
+```sh
+npm i next@canary
+```
+
+or
+
+```sh
+yarn add next@canary
+```
+
+or
+
+```sh
+pnpm add next@canary
+```
+
+If you are using a stable release, upgrade Next.js with one of the following:
+
+```sh
+npm i next@latest
+```
+
+or
```

Comment:
```suggestion
```
We don't use or between the different install variations in examples so we probably shouldn't here either.